### DISCUSSION TITLE
Instapaper 加星后自动保存到简悦的稍后读（包含快照）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#3034] [服务]
> 如果你是 IFTTT 用户，可以利用 IFTTT 和 简悦 Webhook，将 Like 后的稍后读导入到简悦，优势：

1. 此方式会在你的坚果云（或 Dropbox）生成本地快照。
2. 接入到简悦后，就可以使用简悦的各种「导出大礼包」与更专业的标注服务。

https://user-images.githubusercontent.com/81074/141053287-81e5d8b2-5430-400b-b630-c8ad8d0f0b71.mp4

# 前置知识

> 需要事先完成下面的配置。

1. [如何开通开放平台](https://github.com/Kenshin/simpread/discussions/2757)
2. [API 快照功能](https://github.com/Kenshin/simpread/discussions/2994)
3. [如何让稍后读支持读取本地缓存](https://github.com/Kenshin/simpread/discussions/2146)

# 前置知识（之二）

1. 此方式需要使用 IFTTT 这个自动化工具。
2. 使用简悦的 [Webhook](https://simpread.pro/api/#/api?id=添加-webhook)

# 配置

>  如果你使用过 IFTTT，整个过程不会超过五分钟，否则请根据下图的一步步操作即可。 

> [新建一个 Applet]( https://ifttt.com/create)

![image](https://user-images.githubusercontent.com/81074/141050246-5f3e6627-43dc-4e99-9bb4-f7f891d3032b.png)

![image](https://user-images.githubusercontent.com/81074/141050311-db3c607e-5c7d-4509-b837-d0228d2f99f3.png)

![image](https://user-images.githubusercontent.com/81074/141050449-0e6f50f1-1246-4b3c-a75d-251c73cc929b.png)

![image](https://user-images.githubusercontent.com/81074/141050485-aabdffe9-df8a-4601-a650-e83880e52071.png)

![image](https://user-images.githubusercontent.com/81074/141054250-03150f79-2726-4af5-96f6-4aaf2225d4ea.png)

![image](https://user-images.githubusercontent.com/81074/141050787-329c2509-5c87-4f19-96ae-613a224554e1.png)

![image](https://user-images.githubusercontent.com/81074/141050818-127015f8-f0bd-4a3a-a444-bc9c10d14405.png)

![image](https://user-images.githubusercontent.com/81074/141050875-f466e625-6b9c-4328-9995-f8e0b8b0952e.png)

> 登录 [开放平台](![image](https://user-images.githubusercontent.com/81074/141051209-559b18dd-3f9f-41d2-91d2-03dbdd2af725.png)) 得到你的 `token` 并复制到剪切板

![image](https://user-images.githubusercontent.com/81074/141051189-c5a44a9f-b1c2-4a73-8dbe-832a9f86e4c8.png)

> 将 `https://api-wrap.simpread.pro/api/service/webhook/<token>` 中的 `<token>` 替换为上个步骤得到的 `token`

![image](https://user-images.githubusercontent.com/81074/141051799-1c9c7afa-775d-4e99-b410-b537394477f1.png)

**注意**

1. `URL` 为必选参数
2.  `Title` 为空时，会自动抓取快照。
3.  Body 的内容在下方，可直接粘贴到相应的位置。

```
{
"url":"<<<{{URL}}>>>"
}
```

![image](https://user-images.githubusercontent.com/81074/141051907-9f206081-1668-44b1-b80d-e9c17a17e2c4.png)

![image](https://user-images.githubusercontent.com/81074/141051985-61a8d6b4-291e-4f25-8e64-e91ec7bd94d7.png)

> 完成配置
 
![image](https://user-images.githubusercontent.com/81074/141052036-3f0aad54-987c-4cd2-83b5-1ae0877aad9a.png)

# 关联

- [利用 Airtable 实现属于自己的链接公开分享集 #3003](https://github.com/Kenshin/simpread/discussions/3003)
- [Raindrop.io 加星后自动导入到简悦并生成本地快照 #3021](https://github.com/Kenshin/simpread/discussions/3021)
- [加入 Pocket 后自动保存到简悦的稍后读（包含快照） #3024](https://github.com/Kenshin/simpread/discussions/3024)
- [将简悦变成你的 RSS 阅读器（自动下载快照到本地） #3036](https://github.com/Kenshin/simpread/discussions/3036)
- [保存到 Pinboard.in 同时也保存到简悦（包括本地快照） #3048](https://github.com/Kenshin/simpread/discussions/3048)
- [Inoreader 加星后自动保存到简悦的稍后读以及生成本地快照 #3007](https://github.com/Kenshin/simpread/discussions/3007)
- [订阅 Newsletter 后自动保存到简悦的稍后读 #3088](https://github.com/Kenshin/simpread/discussions/3088)
- [当加入稍后读后，自动发送到你的 Telegram Channel #3092](https://github.com/Kenshin/simpread/discussions/3092)
- [发送当前 URL 到飞书群（包括：加入稍后读或手动触发）#3104](https://github.com/Kenshin/simpread/discussions/3104)
- [保存 RSS 的全文到简悦的稍后读（包括快照） #3186](https://github.com/Kenshin/simpread/discussions/3186)

# 引申

- [[教程\]如何开通开放平台 #2757](https://github.com/Kenshin/simpread/discussions/2757)
- [当在开放平台设置完毕后，如何测试配置是否成功？ #3020](https://github.com/Kenshin/simpread/discussions/3020)
- [授权登录（如何使用第三方登录简悦的高级账户） #1570](https://github.com/Kenshin/simpread/discussions/1570)
- [个人中心 #2959](https://github.com/Kenshin/simpread/discussions/2959)
- [免密码登录 #2974](https://github.com/Kenshin/simpread/discussions/2974)

**Comment by yaogjim** [comment]
![image](https://user-images.githubusercontent.com/3283576/186072272-8a91b78a-8bd3-4cbe-85c9-23cd0545b54d.png)
indx 编号 一直没有变化是什么原因？

详细讨论 https://github.com/Kenshin/simpread/issues/4420
